pkg/cmd/build/build.go

```diff
@@ -239,10 +239,12 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		if len(b.VolumesToInclude) > 0 {
 			b.Image = fmt.Sprintf("%s/%s-%s:%s", targetRegistry, manifestName, svcName, model.OktetoImageTagWithVolumes)
 		}
-
 	}
 
-	file := extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
+	file := ""
```

Comment:
It was like this before: 

`file := b.Dockerfile`

Maybe we should add it before the IF to avoid breaking some case...